Tentacle/Client.swift

```diff
@@ -294,103 +283,104 @@ public final class Client {
-        return fetchOne(.ReleaseByTagName(owner: repository.owner, repository: repository.name, tag: tag))
+        return fetchOne(.releaseByTagName(owner: repository.owner, repository: repository.name, tag: tag))
     }
     
     /// Downloads the indicated release asset to a temporary file, returning the URL to the file on
     /// disk.
     ///
     /// The downloaded file will be deleted after the URL has been sent upon the signal.
-    public func downloadAsset(asset: Release.Asset) -> SignalProducer<NSURL, Error> {
-        return NSURLSession
-            .sharedSession()
-            .downloadFile(NSURLRequest.create(asset.APIURL, credentials, contentType: Client.DownloadContentType))
-            .mapError(Error.NetworkError)
+    public func downloadAsset(_ asset: Release.Asset) -> SignalProducer<URL, Error> {
+        return URLSession
+            .shared
+            .downloadFile(URLRequest.create(asset.apiURL, credentials, contentType: Client.DownloadContentType))
+            .mapError(Error.networkError)
     }
     
     /// Fetch the user with the given login.
-    public func userWithLogin(login: String) -> SignalProducer<(Response, UserInfo), Error> {
-        return fetchOne(.UserInfo(login: login))
+    public func userWithLogin(_ login: String) -> SignalProducer<(Response, UserInfo), Error> {
+        return fetchOne(.userInfo(login: login))
     }
 
     /// Fetch the currently authenticated user
     public func authenticatedUser() -> SignalProducer<(Response, UserInfo), Error> {
-        return fetchOne(.AuthenticatedUser)
+        return fetchOne(.authenticatedUser)
     }
 
-    public func assignedIssues(page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
-        return fetchMany(.AssignedIssues, page: page, pageSize: perPage)
+    public func assignedIssues(_ page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
+        return fetchMany(.assignedIssues, page: page, pageSize: perPage)
     }
 
-    public func issuesInRepository(repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
+    public func issuesInRepository(_ repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
         precondition(repository.server == server)
-        return fetchMany(.IssuesInRepository(owner: repository.owner, repository: repository.name), page: page, pageSize: perPage)
+        return fetchMany(.issuesInRepository(owner: repository.owner, repository: repository.name), page: page, pageSize: perPage)
     }
 
     /// Fetch the comments posted on an issue
-    public func commentsOnIssue(issue: Int, repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Comment]), Error> {
+    public func commentsOnIssue(_ issue: Int, repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Comment]), Error> {
```

Comment:
I consistently used `in` for the second parameter of `Repository` type. How do you think about it?